changelog.txt

```diff
@@ -0,0 +1,3 @@
+feature - Upgraded Angular dependency to support 1.4.x
+changed - Removed PhantomJS dependency from e2e tests (now run using Chrome)
```

Comment:
Both of these `changed` items should be removed. The change log tells people what they need to do to update their code. These both tell people about running the repo locally, which less than 1% of users care about.